src/server/replica.cc

```diff
@@ -81,7 +81,8 @@ error_code Replica::Start(ConnectionContext* cntx) {
   ProactorBase* mythread = ProactorBase::me();
   CHECK(mythread);
 
-  RETURN_ON_ERR(cntx_.SwitchErrorHandler(absl::bind_front(&Replica::DefaultErrorHandler, this)));
+  RETURN_ON_ERR(
```

Comment:
removed redundant "Replica::DefaultErrorHandler"

Reply:
Yeah, it probably appeared when it was refactored